src/openrct2/actions/RideSetStatusAction.cpp

```diff
@@ -66,30 +66,21 @@ GameActions::Result::Ptr RideSetStatusAction::Query() const
         return res;
     }
 
-    if (_status >= RIDE_STATUS_COUNT)
```

Comment:
You will still need this code. We need to verify that the information over the network is valid.

Reply:
I've added this back, using a constexpr variable instead of an extra enum variant.